.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,70 @@
+name: "🕷️ Bug report"
```

```suggestion
name: "Bug report"
```

This was not a "bug" ;)